models/rule_helpers_test.go

```diff
@@ -70,6 +70,36 @@ func TestRules(t *testing.T) {
 			assert.ElementsMatch(t, params.ChannelIDs, rule.ChannelIDs)
 		})
 
+		t.Run("rule without channel and filters", func(t *testing.T) {
+			tx, err := db.Begin()
+			require.NoError(t, err)
+			defer func() {
+				require.NoError(t, tx.Rollback())
+			}()
+
+			q := tx.Querier
+
+			templateName := createTemplate(t, q)
+
+			params := createCreateRuleParams(templateName, "", false)
```

Comment:
the same comment here related to using `true/false` in function calls

Reply:
replaced it with this https://github.com/percona/pmm-managed/pull/609/files#diff-7df81ba599594e0fb11c381bb999cc71b71361ba3a9d852784f04e67c37b223cR38